src/parser/jobs/war/Gauge.js

```diff
@@ -0,0 +1,108 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+//import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+// General actions that give Rage (this is how I'm referring to the Warrior gauge) -- Except Storm's Path, since it's a fringe case that gives +20 instead of 10.
```

Comment:
Hm. If you consolidate this into `[id]: <rage delta>`, it'll cut down on some boiler plate and make maintaining the numbers easier. Something like

```js
const RAGE_ACTIONS = {
	[ACTIONS.MAIM.id]: 10,
	[ACTIONS.STORMS_PATH]: 20,
	[ACTIONS.INFURIATE]: 50,
	// ...
}
```

and then you can consolidate the `_onCast` hook to check if there's a key, and handle adding to gauge and tracking wastage using it if there is one.